staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -77,7 +77,7 @@ type queueSetCompleter struct {
 // locking.
 type queueSet struct {
 	clock                eventclock.Interface
-	estimatedServiceTime float64
+	estimatedServiceTime float64 // seconds
```

Comment:
Better to change the variable name.

Best is to change the type to `time.Duration`, then there's no ambiguity.

Reply:
If it were typed as `time.Duration` then every use would add conversion to seconds.

Reply:
... which is at worst a multiplication. I think that's well worth the clarity until a benchmark proves otherwise.

Reply:
I think a field with "seconds" in its name is totally clear.

Reply:
Also, adding the conversions from Duration to seconds is just noise for the readers and maintainers of this code, as well as at runtime.

Reply:
This field is _not_ used in calculations with time.  Rather, it is used in calculations of `R` --- which is a `float64` accumulator of work, quantified in units of seat-seconds.

Reply:
If we used `time.Duration` as the type everywhere, it would also not need any type conversions. It's a good type for tracking seat-seconds.

I'm not insisting on changing the code, but I won't be convinced that a name is as clear as an appropriate type :)

Reply:
`time.Duration` would be a confusing type for seat-seconds, since they differ in dimensions.

Reply:
We _could_ define
```
type seconds float64
```

That would mean we could use the type to say the units.

Reply:
> ... would be a confusing type for seat-seconds ...

This is true of `type seconds float64` also :) (but I do like that solution, too)

I think we've laid out the options and I'm happy to go with your preference at this point.

Reply:
I was thinking of different types for seconds and for seat-seconds.
But yes, let's leave that for a distinct PR (if any).

Reply:
We also have bigger fish to fry in that area: https://github.com/kubernetes/kubernetes/issues/104831